_component_design/switches.md

```diff
@@ -19,29 +18,80 @@ helix-ui-javascript: false
 
 <section class="static-section" markdown="1">
 
-## Introduction
+<div class="hxRow"  markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+## When to use
+
+Use a switch when a user needs to make a binary decision, such as turning a setting on or off or setting a preference enabling or disabling a device or system.
+
+If your use case is outside of these areas, consider a [radio]({{site.baseurl}}/components/radio-buttons.html),  [checkbox]({{site.baseurl}}/components/checkboxes.html), or [selector strip]({{site.baseurl}}/components/selector-strip.html).
+
+{% endcolumn %}
+
+</div>
+
+</section>
+
+<section class="static-section" markdown="1">
 
 <div class="hxRow"  markdown="1">
 
-{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+## Best practices
+
+Consider the following best practices when you construct a switch:
+
+- Construct the switch so that any change to its state is immediately saved. Use a checkbox or radio if the user performs additional steps for changes to be effective. For example, use a radio or checkbox if the user must click a **Submit** or **Next** button to apply changes.
+- A switch changes state when the user clicks it. Do not construct a switch that changes state when the user *clicks and drags* the knob.
+- For clarity, consider including animation that shows the knob changing position.
+- The option label can only be **on/off** or **yes/no**. For any other options, use a selector title or input description that clearly indicates the impact of both binary states on the system. Ensure that the impact of both binary states on the system is clear.
```

Comment:
remove the last sentence here. it's redundant.

s/he option label can only be **on/off** or **yes/no**. For any other options, use a selector title or input description that clearly indicates the impact of both binary states on the system. Ensure that the impact of both binary states on the system is clear./he option label can only be **on/off** or **yes/no**. For any other options, use a selector title or input description that clearly indicates the impact of both binary states on the system.

Reply:
Done.